nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2749,7 +2853,7 @@ root@kdc:~#
 
 [[system_properties]]
 == System Properties
-The _nifi.properties_ file in the _conf_ directory is the main configuration file for controlling how NiFi runs. This section provides an overview of the properties in this file and includes some notes on how to configure it in a way that will make upgrading easier. *After making changes to this file, restart NiFi in order
+The _nifi.properties_ file in the `conf` directory is the main configuration file for controlling how NiFi runs. This section provides an overview of the properties in this file and includes some notes on how to configure it in a way that will make upgrading easier. *After making changes to this file, restart NiFi in order
```

Comment:
Is the new (or previous) standard that directories are `code formatted`?

Reply:
I saw a mixture of both throughout the doc.  I went with code formatted for the new standard.

Reply:
Understood.